CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## UNRELEASED
+## [1.4.0] 2019-15-07
 
 ### Fixed
 - Fix `value`, `eval()` and `getLegendData()` for binary operations
```

Comment:
Here I think we're missing the new options in GeoJSON source (mixed geometry types...)